src/runtime/Context.cpp

```diff
@@ -46,6 +47,23 @@ void* GlobalVariableAccessCacheItem::operator new(size_t size)
     return GC_MALLOC_EXPLICITLY_TYPED(size, descr);
 }
 
+#ifdef ESCARGOT_DEBUGGER
+
+void* Context::operator new(size_t size)
```

Comment:
I think you can remove this function.
every member variables other than m_debugger are allocated by GC
so we should mark every member variables thus we should not override operator new